tests/basic/basic_test.go

```diff
@@ -487,6 +487,62 @@ var _ = Describe("{CordonDeployDestroy}", func() {
 	})
 })
 
+var _ = Describe("{CordonStorageNodesDeployDestroy}", func() {
+	var contexts []*scheduler.Context
+
+	It("has to cordon all storage nodes, deploy and destroy app", func() {
+
+		Step("Cordon all storage nodes", func() {
```

Comment:
Probably better to skip the test if there are no storage less nodes?

Reply:
done.